README.md

```diff
@@ -1,2 +1,41 @@
 # Portfolio
-SergSkol Portfolio
+
+> There is SergSkol Portfolio (Module 1 Day 3)
+
+## Built With
+
+- HTML, CSS
+- Visual Studio Code
+- GIT and GITHUB
+
+## Live Demo (if available)
+
+https://sergskol.github.io/portfolio
+
+## Getting Started
+
+Follow the link to open web page.
+
+## Authors
```

Comment:
- [OPTIONAL] Kindly change this to `Author`. This is a solo project and every solo project carries only one author.

Reply:
Thank you! Done!